### DISCUSSION TITLE
Add support for setting WhatsApp status

### DISCUSSION THREAD

**Discussion by hellocodelinux** [d99kris/nchat#295] [Ideas]
First of all, I thank you from the bottom of my heart for nchat, as my low-resource PC cannot handle other versions!! Thanks, thanks, thanks.

While it allows sending statuses, they are not visible to other users. Both the web and the app report that it was sent with an invalid system.

**Comment by d99kris** [comment]
Hi @hellocodelinux - thanks for the positive feedback! 👍 

Currently nchat does not support sending/setting WhatsApp statuses AFAIK. Just to clarify, are you talking about WhatsApp status, or WhatsApp broadcast messages?

Anyway, I'm changing the discussion to a feature request, so we can gauge other users interest and track possible implementation.
